radsat-sk/operation/subsystems/RDosimeter.c

```diff
@@ -203,9 +204,9 @@ int16_t dosimeterTemperature(dosimeterBoard_t board) {
 						DOSIMETER_RESPONSE_LENGTH, &dosimeterCommandBytes[adcChannel],
 						dataResponse, DOSIMETER_I2C_DELAY);
 
-	// return 0 if an error occurs
+	// return 1 if an error occurs
 	if (error != 0)

```

Comment:
```suggestion
	if (error != SUCCESS)

```

This change should also be made in the other files as on line 144 in this file. be carful not to change the logic.